kubernetes-deploy.gemspec

```diff
@@ -25,14 +25,14 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '>= 2.4.0'
   spec.add_dependency("activesupport", ">= 5.0")
   spec.add_dependency("kubeclient", "~> 4.3")
-  spec.add_dependency("googleauth", "~> 0.8.0")
+  spec.add_dependency("googleauth", "~> 0.8")
```

Comment:
@dirceu I made a slight adjustment to match the googleauth dependency restriction with Krane's latest. You still 👍 ?